ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/WebSocketMessageHandler.java

```diff
@@ -172,4 +178,12 @@ private void replyToClient(final ServerWebSocket websocket, final Object result)
   private JsonRpcResponse errorResponse(final Object id, final RpcErrorType error) {
     return new JsonRpcErrorResponse(id, error);
   }
+
+  private void traceResponse(final Object response) {
+    try {
+      LOG.trace(jsonObjectMapper.writeValueAsString(response));
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException(e);
```

Comment:
do we want to throw if we can't log? where does this exception come out?

Reply:
Yeah I think you're right, logging it is safer. We do have places where we throw a runtime exception for the same JSON-parsing issue e.g. https://github.com/hyperledger/besu/blob/7770f966c92c638b7a6233a22d3fdd4f7a27f1ca/ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineExchangeTransitionConfiguration.java#L80

Perhaps that should be a log as well, instead of a runtime exception?

Reply:
yeah I'd say the same thing for there